contracts/utils/fileSystem.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const papa = require("papaparse");
+const { ethers } = require("hardhat");
+
+
+const parseCsv = (filePath) => {
```

Comment:
shouldn't this be async since it returns a Promise?

Reply:
👍